server/channels/app/email/email_batching.go

```diff
@@ -278,6 +279,13 @@ func (es *Service) sendBatchedEmailNotification(userID string, notifications []*
 		}
 	}
 
+	var useMilitaryTime bool
+	if data, err := es.store.Preference().Get(user.Id, model.PreferenceCategoryDisplaySettings, model.PreferenceNameUseMilitaryTime); err != nil {
+		useMilitaryTime = true
```

Comment:
This should be false since we default to 12 hour time elsewhere in the app

Reply:
Do you know where it defaults to false? I referenced [the non batched email code](https://github.com/mattermost/mattermost/blob/f130615011164c6d184176e3bad995bacf147f9d/server/channels/app/notification_email.go#L77) and it defaults to true there.  If incorrect there probably worth a separate issue as well, and possibly refactor that logic into utils as well.

Reply:
addressed in [8121af7](https://github.com/mattermost/mattermost/pull/26121/commits/8121af7241892c32579360af0f1a90baf3b14438)

Reply:
Sorry for the lack of response earlier. I didn't realize that that was copied from elsewhere. I think that's actually a bug then since that `Get` method returns an error if the setting doesn't exist, and everywhere on the frontend defaults that value to false. Here's an example if you're curious.

https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/components/timestamp/index.ts#L32

I'll file a followup ticket to address that since, while it's related to the changes here, it doesn't need to block this from being added